components/inputs/demo/input-time.html

```diff
@@ -53,6 +53,29 @@ <h2>Custom Default Time</h2>
 					<d2l-input-time label="Start Time" default-value="09:00:00"></d2l-input-time>
 				</template>
 			</d2l-demo-snippet>
+
+			<h2>Inline help text</h2>
+			<d2l-demo-snippet>
+				<template>
+					<d2l-input-time label="Start Time" default-value="09:00:00">
```

Comment:
Is this what we expect for `d2l-input-time`? Initially I'd expected it to look like the others, but then discovered that we're actively limiting the width, so maybe this is ok. @margaree might have more context here.
![image](https://github.com/BrightspaceUI/core/assets/9042472/164aa48b-d362-4c82-81ca-2e87c43cec5d)

Reply:
This PR https://github.com/BrightspaceUI/core/pull/1089/files is context on why we are limiting the width. We could possibly move that calculation into `input-date-time` since that's where that problem was but I'm not sure if there are places now relying on its size being as small as possible by default.

Reply:
When I moved the max-width to the dropdown or altered it the component looped around itself. The label text was in the same line as the input text